docs/ref/models/database-functions.txt

```diff
@@ -149,6 +149,39 @@ will result in a database error.
     The PostgreSQL behavior can be emulated using ``Coalesce`` if you know
     a sensible maximum value to provide as a default.
 
+``NullIf``
+----------
+
+.. class:: NullIf(expression1, expression2)
+
+.. versionadded:: 2.1
+
+Accepts two arguments, and returns ``None`` if they are equal, otherwise
+returns ``expression1``.
+
+.. note:: Caveats when using Oracle
+
+    Due to convention, this function returns empty string instead of ``None``.
```

Comment:
I think we should mention that this applies only to `CharField`s.

Reply:
Surely `TextField` also?

Reply:
`IFNULL` will not work with `TextField` (`NCLOB`) from the same reason as for `BinaryField` (`BLOB`) 😒, see [documentation](https://docs.oracle.com/database/121/ADLOB/adlob_sql_semantics.htm#GUID-D8F66A2A-4D17-49C3-ADB2-BE384510DD6D).